internal/cli/cli.go

```diff
@@ -0,0 +1,24 @@
+package cli
```

Comment:
If folder has name **internal**, that means, that there should be and **external** one

imho, **lib** or **sources** is more correct naming